xml/docu_styleguide.terminology.xml

```diff
@@ -1228,6 +1228,16 @@
       <entry>R.O., RO, read only, readonly, ro</entry>
       <entry>adjective</entry>
      </row>
+     <row>
+      <entry>real time</entry>
+      <entry>real-time</entry>
+      <entry>noun; e.g. <quote>watch in real time</quote></entry>
```

Comment:
```suggestion
      <entry>noun; as in: “watch in real time”</entry>
```

Reply:
I've applied this manually without the quotation marks.